recipes/cpplint/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cpplint" %}
+{% set version = "1.4.4" %}
+{% set sha256 = "05f879aab5a04307e916e32afb547567d8a44149ddc2f91bf846ce2650ce6d7d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/cpplint/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    noarch: python
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
```

Comment:
You need to add `entry_points` for this to work as a `noarch` package:
* https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html?highlight=selectors#python-entry-points
* https://github.com/cpplint/cpplint/blob/master/setup.py#L62-L66

Reply:
Thanks. It works now.